src/main/resources/assets/spruceui/lang/it_it.json

```diff
@@ -1,11 +1,12 @@
 {
-  "spruceui.reset": "Ripristina",
-  "spruceui.gui.unbind": "Disassocia",
-  "spruceui.narrator.separator": "Separatore %s",
+  "spruceui.reset": "Resetta",
+  "spruceui.gui.unbind": "Disattiva",
```

Comment:
"Dissocia" is probably a better word for this, as "Disattiva" implies an on/off state, while unbind does not ( and has nothing to do with it )